.travis.yml

```diff
@@ -3,6 +3,9 @@ language: crystal
 before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y ruby default-jdk haskell-platform python3
+  - wget http://www.lua.org/ftp/lua-5.3.4.tar.gz
+  - tar -zxv -f lua-5.3.4.tar.gz
+  - pushd lua-5.3.4 && make CFLAGS="-O2 -fPIC -DLUA_COMPAT_5_2 -DLUA_COMPAT_5_1" linux && sudo make INSTALL_TOP=/usr install && popd
```

Comment:
Why not install lua with apt?

Reply:
It requires lua5.3, and there are only lua5.0, lua 5.1, and lua5.2 in apt.